ci/build-all-steps.yml

```diff
@@ -84,14 +88,14 @@ steps:
       call install\local_setup.bat
       cd \dev\ros2
       vcs branch
-      ament build ^
-        --parallel ^
+      colcon build ^
+        --merge-install ^
         --cmake-args ^
-          -G "${{ parameters.generator }}" ^
+          -A "${{ parameters.arch }}" ^
+          -DBUILD_TESTING=OFF ^
           -DTHIRDPARTY=ON ^
           -DINSTALL_EXAMPLES=OFF ^
-          -DCMAKE_FIND_ROOT_PATH="\dev\ament\install;\dev\ros2\install" ^
-        --
+          -DCMAKE_FIND_ROOT_PATH="\dev\ament\install;\dev\ros2\install"
```

Comment:
I think this shouldn't be necessary with colcon. I haven't needed it so far, at least.

Reply:
Removed